include/swigpytest.h

```diff
@@ -0,0 +1,105 @@
+//  Copyright (c) 2021-23, VMware Inc, and the Certifier Authors.  All rights
+//  reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// ****************************************************************************
+// These are just dummy class definitions, based on the stuff defined in
+// certifier_framework.h . These definitions and interfaces are used solely:
+// - to exercise different combinations of SWIG-interface rules
+// - to verify via pytests the way to exercise these interfaces
+// - to standardize on code structure to exchange data from Python code to C++
```

Comment:
This comment block should help you understand why I am adding this test-only .h file.

We need to discuss the "right" place for this .h file. Currently, it's simple for the Make-rules to find this in the `include/` dir, but that may not be the ideal place.

I don't want to "fix" the right location right now ... Let's wait for all this bindings work to stabilize and then we can re-work the final dir-location for this file (is my thinking).